### PR TITLE
Home video listing

### LINKED ISSUE #2: Epic: Video listing page
# Video listing page

Feature: Video listing

## Epic Description

This Feature will 

## List of Tasks (Complete in order)

- [ ] 1. Show all available videoes
- [ ] 2. Add videos context
- [ ] 3. Add video categories
- [ ] 4. Create single video card
- [ ] 5. Show filtered videos by categories

### LINKED ISSUE #4: Epic: Home page
# Epic Title

Feature: Home page

## Epic Description

This Feature will complete the home page

## List of Tasks (Complete in order)

1. [ ] Create categories context
2. [ ] Add categories context
3. [ ] Add categories chips container
4. [ ] Add categories filter

### PR DESCRIPTION
**Pull request by Pratham82**
# Story Title

[Epic: Home page #4](https://github.com/Pratham82/video-lib/issues/4)
[Epic: Video listing page #2](https://github.com/Pratham82/video-lib/issues/2)

## Changes made

- [x] Completed homepage/video listing page

## How does the solution address the problem

This PR will complete the homepage/ video listing page epics.

## Linked issues

Resolves #2, Resolves #4

### CODE REVIEW

**Comment by ashifkhn** [comment]
UI looks great. Good to merge.

File: src/contexts/videos/videoState.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      }
+    })();
+  }, []);
+
+  const { videos, selectedCategories, categories, loading } = videosData;
+  const filteredVideos = getFilteredVideos(videos, selectedCategories);
+
+  return (
+    <VideoContext.Provider
+      // eslint-disable-next-line react/jsx-no-constructed-context-values
```
Comment (Balakumar3018) [inline src/contexts/videos/videoState.tsx:44]: remove commented code

**Review by Balakumar3018 — APPROVED** [review]
Functionality is working as expected. Good to merge

File: src/contexts/videos/videoState.tsx (hunk 1 above)
Comment (Pratham82) [reply src/contexts/videos/videoState.tsx:44]: It's actually used for fixing es-linting warnings.

**Pull request merged**